MH-FiorellaTheme5/source/stylesheets/styles.css

```diff
@@ -3,37 +3,49 @@ html, body {
   margin: 0;
   padding: 0;
 }
-
 p {
   text-align: center;
 }
-
 .header {
-  display: flex;
-  justify-content: center;
-  align-items: flex-start;
-  width: auto;
-  margin:0;
-  height: 700px;
-  background-image: url("../images/home/header_background.jpg");
-  background-size: 100%;
-  background-repeat: no-repeat;
+  width: 100%;
 }
-
 .upper-row {
   background-color: #F3F9EF;
   opacity: 0.5;
   height: 50px;
   width: auto;
 }
-
+/* Buttons */
+.btn-slider,.btn-call-action{
+  font-family: 'PlayfairDisplay-Bold';
+  background: transparent;
+  color: #E83671;
+  border: 2px solid #E83671;
+  font-weight: bold;
+}
+/* end buttos */
+/* slider */
+.img-item{
+   position: relative;
+   text-align: right;
+}
+.img-item img{
+  width: 100%;
+  height: auto;
+}
+.msg{
+  letter-spacing: 1px;
+}
+.slick-next .slick-arrow{
+  display: none;
+}
+/* end slider */
```

Comment:
No need to add comments, your classes should be the in order of how your view is.